.github/workflows/e2e-tests.yml

```diff
@@ -95,42 +98,6 @@ jobs:
         working-directory: grafana-plugin
         run: yarn build:dev
 
-      - name: Set up Docker Buildx # We need this step for docker caching
```

Comment:
@brojd I suppose the Tiltfile will already take care of building the backend Docker image?

Reply:
Yeah, we have this part in Tiltfile:
```
# Build the image including frontend folder for pytest
docker_build_sub(
    "localhost:63628/oncall/engine:dev",
    context="./engine",
    cache_from=["grafana/oncall:latest", "grafana/oncall:dev"],
    ignore=["./test-results/", "./grafana-plugin/dist/", "./grafana-plugin/e2e-tests/"],
    child_context=".",
    target="dev",
    extra_cmds=["ADD ./grafana-plugin/src/plugin.json /etc/grafana-plugin/src/plugin.json"],
    live_update=[
        sync("./engine/", "/etc/app"),
        run(
            "cd /etc/app && pip install pip-tools && pip-sync",
            trigger="./engine/requirements.txt",
        ),
    ],
)
```